pages/common/az-account.md

```diff
@@ -0,0 +1,24 @@
+# az account
+
+> Manage Azure subscription information.
+> More information: <https://docs.microsoft.com/cli/azure/account>.
+
+- Print a list of subscriptions for the logged in account:
+
+`az account list`
+
+- Set a `subscription` to be the currently active subscription:
+
+`az account set --subscription {{subscription_id}}`
+
+- List supported regions for the currently active subscription:
+
+`az account list-locations`
+
+- Print an access token to be used with `MS Graph API`:
+
+`az account get-access-token --resource-type {{ms-graph}}`
+
+- Print details of the currently active subscription in the specified format:
```

Comment:
```suggestion
- Print details of the currently active subscription in a specific format:
```

Reply:
`the specific` is what we use in most pages, even tough a little variety doesn't hurt.

Reply:
I still feel that `a` is required, and with an `a`, `specific` reads nicer